internal/httputil/auth.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package httputil
+
+import (
+	"net/http"
+	"strings"
+)
+
+// FormatAuthString reads the value of request Authorization and removes the Bearer substring
```

Comment:
Why do we do this? Why don't we just change API and demand that the new confention is to send `Bearer` in Autorisation header?

Reply:
I also agree. @mteodor I understand that this is for integration, and we should add it. However, since this is the standard, let's switch to it and require a prefix. That's all. At least when it comes to the API for users, we don't have to change the HTTP adapter for things in this PR. Also, this implies changes in SDK, tests, and docs.